console/src/components/Services/EventTrigger/Add/AddState.js

```diff
@@ -6,7 +6,7 @@ const defaultState = {
   operations: { insert: [], update: [], delete: [] },
   selectedOperations: { insert: false, update: false, delete: false },
   webhookURL: '',
-  webhookUrlType: 'url',
+  webhookUrlType: 'env',
```

Comment:
The default option is intentionally made to be `url` as it will serve most of the use cases unless user would like to enter something `secret`. Could you also explain what is the motivation behind this?